doc/basics/discoverystrategy_tutorial.rst

```diff
@@ -0,0 +1,56 @@
+Network IO and the DiscoveryStrategy
+====================================
+
+This document assumes you have a basic understanding of asyncio tasks, as documented in `the tasks tutorial <../../basics/tasks_tutorial>`_.
+You will learn how to use the IPv8's ``DiscoveryStrategy`` class to avoid network congestion.
+
+The DiscoveryStrategy
+---------------------
```

Comment:
@devos50 These lines and the next section heading are new since your last review.